resources/ts/Components/Navigation.tsx

```diff
@@ -37,9 +37,7 @@ export default function Navigation({
   const { t, i18n } = useTranslation();
   const translateRoute = useTranslateRoute();
 
-  const path = new URL(document.location.href).pathname;
-
-  const pages = ["integrations", "support"];
+  const pages = ["integrations", "integrations/new", "support"];
 
   const { component } = usePage();
```

Comment:
Can't we just usse this to infer the current page and compare it to the links instead of keeping active link in state?

Reply:
Since we use the startsWith() method and both strings start with "integrations", it doesn't seem to be working.